packages/widget-utils/src/bootstrap/bootstrapWidgets.js

```diff
@@ -5,22 +5,29 @@ const bootstrapWidgets = async params => {
 
   const widgetContainerNodeList = document.querySelectorAll('[data-tocco-widget]')
   const widgetContainers = Array.prototype.slice.call(widgetContainerNodeList)
-  const apps = [...new Set(widgetContainers.map(container => container.getAttribute('data-tocco-widget')))]
+  const packages = [
+    ...new Set(
+      widgetContainers.map(
+        container => container.getAttribute('data-tocco-package') || container.getAttribute('data-tocco-widget')
+      )
+    )
+  ]
 
   await Promise.all(
-    apps.map(app => {
-      return loadScriptAsync(`${backendUrl}/js/tocco-${app}/dist/index.js`)
+    packages.map(packageName => {
+      return loadScriptAsync(`${backendUrl}/js/tocco-${packageName}/dist/index.js`)
     })
   )
 
   widgetContainers.forEach(container => {
     const app = container.getAttribute('data-tocco-widget')
+    const packageName = container.getAttribute('data-tocco-package') || app
```

Comment:
Small detail... I would prefer to have a small function to get the package name.
This could be used on line 11 as well